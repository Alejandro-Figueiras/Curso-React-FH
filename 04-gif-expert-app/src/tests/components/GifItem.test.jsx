import { describe, expect, test } from "vitest"
import { render } from '@testing-library/react';
import GifItem from '../../components/GifItem';

describe("Pruebas en <GifItem />", () => {
  const title = 'Hola';
  const url = 'https://localhost/algo.jpg';

	test("should match the snapshot", () => {
    const {container} = render(<GifItem title={title} url={url} />)

    expect(container).toMatchSnapshot();
  })
})
