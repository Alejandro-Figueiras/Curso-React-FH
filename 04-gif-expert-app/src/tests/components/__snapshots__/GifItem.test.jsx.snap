// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Pruebas en <GifItem /> > should match the snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="Hola"
      src="https://localhost/algo.jpg"
    />
    <p>
      Hola
    </p>
  </div>
</div>
`;
